masonry_core/src/doc/04_testing_widget.md

```diff
@@ -89,9 +89,9 @@ Let's add a visual test:
 
 The [`assert_render_snapshot!`] macro takes a snapshot name, renders the current state of the app, and stores the render as a PNG next to the test, in a `./screenshots/` folder.
 
-The rendered screenshot is compared against an existing file checked in your project, and panics if the reference file is meaningfully different (using [the NVIDIA FLIP algorithm](https://developer.nvidia.com/blog/flip-a-difference-evaluator-for-alternating-images/)) or if there isn't one.
+The rendered screenshot is compared against an existing file checked in your project, and panics if the reference file is meaningfully different (with some tolerance for small pixel-by-pixel differences) or if there isn't one.
```

Comment:
Is it no longer using that algorithm? If it is still using it, I am not sure I'd want the reference removed unless it is duplicate.

Reply:
Nope, the dependency to `nv-flip` was removed in #904.